web/bfex/blueprints/search_api.py

```diff
@@ -26,14 +27,17 @@ def get(self):
         pf_query = q_parser.parse_query(query)
         elastic_query = q_builder.build(pf_query)
 
+        allowed_dept = ConfigFile().data['departments']
```

Comment:
I wouldn't make this a configuration file. You should just be able to pass in the filtered departments as a url parameter. Otherwise you are completely limiting anyone from getting that info.

Reply:
Hey gary, can you handle this one for me? I've been struggling on this for a while since I don't understand query builder as well as you do. I've tried writing custom multi_match, bool, match and didn't work out, but I also don't want to modify your query builder code.